model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/entities/AuthenticatedClientSessionEntity.java

```diff
@@ -42,6 +44,7 @@ public class AuthenticatedClientSessionEntity extends SessionEntity {
 
     // Metadata attribute, which contains the last timestamp available on remoteCache. Used in decide whether we need to write to remoteCache (DC) or not
     public static final String LAST_TIMESTAMP_REMOTE = "lstr";
+    public static final String USER_SESSION_STARTED_AT_NOTE = "userSessionStartedAt";
```

Comment:
This needs to be in `AuthenticatedClientSessionModel`, not `AuthenticatedClientSessionEntity`

Reply:
OK! It's done now! I have also created the method `getUserSessionStarted` more or less the same to `getStarted`.